pages/staff/academic_support.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: default
-permalink: /staff/academic_support_staff/
+permalink: /staff/academic-support-staff/
```

Comment:
These link changes broke a lot of things 🤣
These are used by the search jsons too